views/vm/create.erb

```diff
@@ -1,5 +1,20 @@
 <% @page_title = "Create Virtual Machine" %>
 
+<% unless @has_valid_payment_method %>
+  <div class="-mt-6 mb-2 rounded-md bg-red-50 p-4">
+    <div class="flex">
+      <div class="flex-shrink-0">
+        <%== render("components/icon", locals: { name: "hero-x-circle-mini", classes: "h-5 w-5 text-red-400" }) %>
+      </div>
+      <div class="ml-3">
```

Comment:
Project doesn't have valid billing information. You have to **create** project's billing details first.

Click here to **create** billing details.

Reply:
<img width="1512" alt="Screenshot 2023-08-11 at 08 43 33" src="https://github.com/ubicloud/ubicloud/assets/993199/7ee87a2b-a9dc-45d5-a917-6e4c0cdcc635">